space-age/SpaceAgeTest.m

```diff
@@ -0,0 +1,70 @@
+#import <XCTest/XCTest.h>
+#import "SpaceAge.h"
+
+@interface test_suite : XCTestCase
+
+- (void)assertValue1:(double)value1 isEqualToValue2:(double)value2;
+
+@end
+
+@implementation test_suite
+
+- (void)assertValue1:(double)value1 isEqualToValue2:(double)value2 {
+    double accuracyLevel = 0.01;
+    XCTAssertEqualWithAccuracy(value1, value2, accuracyLevel);
+}
+
+- (void)testAgeInSeconds {
+    SpaceAge *age = [[SpaceAge alloc] initWithSeconds:1000000];
+    XCTAssertEqual(1000000, [age seconds]);
+    
+}
+
+- (void)testAgeInEarthYears {
+    SpaceAge *age = [[SpaceAge alloc] initWithSeconds:1000000000];
+    [self assertValue1:31.69 isEqualToValue2:[age onEarth]];
```

Comment:
Instead of creating a helper method, how about defining a constant for the accuracy value?
eg: `static double AgeAccuracy = 0.01;` at the top level.  Then using that constant in a call to XCTAssertEqualWithAccuracy in each test.